ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConstants.java

```diff
@@ -44,7 +49,24 @@ private KsqlConstants() {
   public static final String AVRO_SCHEMA_NAMESPACE = "io.confluent.ksql.avro_schemas";
   public static final String AVRO_SCHEMA_NAME = "KsqlDataSourceSchema";
   public static final String DEFAULT_AVRO_SCHEMA_FULL_NAME =
-          AVRO_SCHEMA_NAMESPACE + "." + AVRO_SCHEMA_NAME;
+      AVRO_SCHEMA_NAMESPACE + "." + AVRO_SCHEMA_NAME;
+
+  public enum KsqlQueryState {
```

Comment:
`KsqlQueryState` makes it sounds like its referring to a state store or underlying state.. May be `KsqlQueryStatus` is a better name? 

Just a thought. Leave it to you :)

Reply:
hmm, I see what you mean and I agree. I used state here because we previously had `getState` for the QueryMetadata object so I kind of just continued using State in the recent changes I made and this current one.

I'm not sure exactly what we want to be calling this, should we just make the change in this PR to try to use `status` wherever we can instead of `state`?

Reply:
@vinothchandar I think I'm leaning more on the side of just keeping it state for now since I'm keeping the `state` field in the RunningQuery and QueryDescription response objects for backwards compatibility.

Any thoughts @rodesai ? The upcoming query error handling would be using this enum.

Reply:
I vote for status to disambiguate from queryable state.

Reply:
error and status go more hand in hand IMO.. I do see this is "Streams' state" per se. leave it to you :)

Reply:
I've changed it to `KsqlQueryStatus` now